### DISCUSSION TITLE
Use coc.nvim on iPhone

### DISCUSSION THREAD

**Discussion by fannheyward** [neoclide/coc.nvim#2699] [Show and tell]
Not an issue 😆 

![image](https://user-images.githubusercontent.com/345274/101780623-bc171e00-3b31-11eb-861b-ca0bfd46965f.png)
 
- iSH
- `apk add vim nodejs npm`

**Comment by oblitum** [comment]
wow

**Comment by iamcco** [comment]
Android and termux:

![image](https://user-images.githubusercontent.com/5492542/103131385-82bae280-46db-11eb-860a-ce00aa638fcd.png)
